### DISCUSSION TITLE
Improved semantic graph and LLM-powered search (question-answering)

### DISCUSSION THREAD

**Discussion by anngvu** [nf-osi/research#25] [Ideas]
What kind of question would the community most want to enable through better search?

**Comment by anngvu** [comment]
Has this mutation shown up in schwannoma or plexiforme neurofibroma?

**Comment by anngvu** [comment]
Are there any chemical compounds that look effective for XYZ cell line?

**Comment by anngvu** [comment]
What studies/data should I look at if I want to reuse data to study XYZ or talk to people who study XYZ -- based on provided abstract/description, give ranked list that personalizes results.

**Reply by anngvu** [reply]
Using Sasha's project as a test to see how well this works

**Reply by allaway** [reply]
note to self: integrate multiple data sources, publications data, uploaded metadata

will flesh this out later

**Comment by anngvu** [comment]
What tools (assays, models, etc.) have been used/made available by others that can help me design my own study that needs to look at [pain behavior, a knockout of this variant, other condition]?

**Reply by anngvu** [reply]
Especially for assays, want to be able to incorporate the descriptions and semantic meaning painstakingly maintained in the data model but not used at all in portal search at all.

**Reply by allaway** [reply]
Could also incorporate ontology information from source uri (e.g. parent info, ontology context)

**Comment by anngvu** [comment]
Questions related to data sharing governance/provenance, which might be more accessible through natural language queries than reading the docs. For example: What is the preferred open format for XYZ data type according to the NF community guidelines?

**Comment by anngvu** [comment]
Find data related to X phenotype.

**Comment by anngvu** [comment]
Layperson, more exploratory questions?
